docs/containers-signature.5.md

```diff
@@ -135,7 +135,7 @@ Consumers MUST reject any signature where the `critical` object contains any unr
 
 ### `critical.type`
 
-This MUST be a string with a string value exactly equal to `atomic container signature` (three words, including the spaces).
+This MUST be a string with a string value exactly equal to `container signature` (three words, including the spaces).
```

Comment:
This would immediately break all existing signatures, and is completely impractical.

Introducing a new value, and figuring out some migration plan (still breaking old consumers), is IMHO not at all worth it.

Reply:
Removed change.